test/avro_union_tests.erl

```diff
@@ -1,10 +1,23 @@
 %%%-------------------------------------------------------------------
-%%% @author tihon
-%%% @copyright (C) 2016, Klarna AB
-%%% @doc
+%%% Copyright (c) 2013-2016 Klarna AB
 %%%
+%%% This file is provided to you under the Apache License,
+%%% Version 2.0 (the "License"); you may not use this file
+%%% except in compliance with the License.  You may obtain
+%%% a copy of the License at
+%%%
+%%%   http://www.apache.org/licenses/LICENSE-2.0
+%%%
+%%% Unless required by applicable law or agreed to in writing,
+%%% software distributed under the License is distributed on an
+%%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%%% KIND, either express or implied.  See the License for the
+%%% specific language governing permissions and limitations
+%%% under the License.
+%%%
+%%% @author Ilya Staheev <ilya.staheev@klarna.com>
+%%% @doc Avro Json decoder
```

Comment:
please fix the docs for all test modules (simply remove is good enough I suppose)